client/src/ui/product/ZoomButtons.vue

```diff
@@ -6,31 +6,29 @@
     scale,
     MAX_SCALE,
     MIN_SCALE,
+    setScale,
+    DEFAULT_SCALE_JUMP,
   } from '@utils/components/usePinchToZoom';
   import GWell from '@ui/graph/GWell.vue';
-
-  const setScale = (scaleChange: number) => {
-    scale.value = Math.round((scale.value + scaleChange) * 10) / 10;
-  };
 </script>
 
 <template>
   <GToolbarBase>
     <ToolbarButtonGroup>
       <GToolbarButton
-        @click="setScale(0.1)"
-        :disabled="scale >= MAX_SCALE"
-        icon="plus"
+        @click="setScale(-DEFAULT_SCALE_JUMP)"
```

Comment:
what if the zoom api exported helpers like zoomIn and zoomOut which already came prebaked with the defaults so you never have to worry about passing this value in on each invocation